### DISCUSSION TITLE
Windows Intel Emulation Performance on M1 Ultra?

### DISCUSSION THREAD

**Discussion by hstriepe** [utmapp/UTM#3868] [UTM Help (macOS)]
Has anybody tried Windows Intel emulation on an M1 Ultra? Is it usable?

In the past I found it excruciating on an M1 MacBook Pro.

**Comment by Belg0** [comment]
I just tried it on a M1 Mac Mini. It was almost impossible to do, and when I finally got it to work. It was indeed excruciating slow.
